app/google-services.json

```diff
@@ -0,0 +1,68 @@
+{
+  "project_info": {
+    "project_number": "815978434764",
+    "project_id": "jetpack-compose-crypto",
+    "storage_bucket": "jetpack-compose-crypto.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:815978434764:android:2c81917e30236ed4fc7269",
+        "android_client_info": {
+          "package_name": "co.nimblehq.compose.crypto"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "815978434764-h4p3pe9e9198t33ghiv2ptp99b0nqko5.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBPYTu8leHXVj8iqmnrSyMllEm0Ckbp8t4"
+        }
+      ],
+      "services": {
+        "appinvite_service": {
+          "other_platform_oauth_client": [
+            {
+              "client_id": "815978434764-h4p3pe9e9198t33ghiv2ptp99b0nqko5.apps.googleusercontent.com",
+              "client_type": 3
+            }
+          ]
+        }
+      }
+    },
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:815978434764:android:fabffa0f76e72b60fc7269",
+        "android_client_info": {
+          "package_name": "co.nimblehq.compose.crypto.staging"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "815978434764-h4p3pe9e9198t33ghiv2ptp99b0nqko5.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBPYTu8leHXVj8iqmnrSyMllEm0Ckbp8t4"
+        }
+      ],
+      "services": {
+        "appinvite_service": {
+          "other_platform_oauth_client": [
+            {
+              "client_id": "815978434764-h4p3pe9e9198t33ghiv2ptp99b0nqko5.apps.googleusercontent.com",
+              "client_type": 3
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "configuration_version": "1"
+}
```

Comment:
End of file.

Reply:
@Wadeewee not a big deal on this file I think 😛